Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 4
+version = 3
```

Comment:
Isn't it enough to change the version of the Cargo.lock file and keep the dependencies at their latest versions?

(I.e. keep the change above and revert the rest.)

Reply:
You're right. I was hesitant to do this by hand so I had it deleted and tried to generate a new file, but I was getting errors when trying to compile some dependencies. 

I have reverted Cargo files back to version from main, downgraded Cargo.lock version to `3` manually and had the runtime package version bumped in commit https://github.com/CheckerNetwork/zinnia/pull/691/commits/b1458e344bc9f69bbfc84f1f531f7442f884b59a